core/domain/event_services_test.py

```diff
@@ -238,6 +238,8 @@ def _get_all_python_files(self):
             if not (directory.startswith('./core/') or
                     directory.startswith('./extensions/')):
                 continue
+            if directory.startswith('./core/tests/linter_tests'):
```

Comment:
Done.

Reply:
This (as well as the other two files) search for all the python files in core/ or extensions/, so we have to ignore linter_tests since it contains "mock" python files used for testing.